cvrf2csaf/cvrf2csaf.py

```diff
@@ -170,6 +176,10 @@ def main():
     # Update & rewrite config file values with the ones from command line arguments
     config.update(args)
 
+    # Boolean optional argument need special treatment
+    if config['force'] == 'forced-conversion':
```

Comment:
I may be wrong trying to follow on a tablet per GitHub web ui, but this can still be an empty dict, right? In that case a KeyError may result from the condition evaluation.

Also, do we really want to replace a string with a boolean in case it exactly matches? If there is a text `foo`  coming  from the YAML configuration the subsequent boolean evaluation of that dict key will also be true. Only the `is True` comparison will fail for the latter.

Reply:
@sthagen - I see your point. 

Actually, as long as `force` is not removed from the config file this will not raise a key error. But I agree, the `.get()` is of course a more robust way to access it.

For the second point,  `forced-conversion` as a value is not ideal, if you combine CLI options + config values.

Integrated `store_true` instead [here](https://github.com/csaf-tools/CVRF-CSAF-Converter/commit/50537143b955c9e48c3ae1aca29da47abc578486#diff-fa1a64ee695c90205cb0dc8aba4aa7a0dc8eb5e70ede667636ade192a8ddaaa1R152). This should solve both.